backend/services/FileHandler.py

```
@@ -0,0 +1,45 @@
+from schemas.post import PostResponse, MediaPostResponse, DocumentPostResponse, EventResponse, TextPostUpdate
+from database.session import SessionLocal
+from zoneinfo import ZoneInfo
+from sqlalchemy.orm import Session, joinedload
+import shutil
+from core.connection_crud import get_connections
+from crud.notification import create_notification
+from AI.moderation import moderate_text
+
+STATUS_404_ERROR = "Post not found"
+
+def validate_file_extension(filename: str, allowed_extensions: set):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_file_extension`

(ANN201)